addPost.php

```diff
@@ -0,0 +1,134 @@
+<?php
+    include('include/init.php');
+
+    if(isset($_POST['postItem'])) {
+        $postingTitle = $_POST['postingTitle'];
+        $price = $_POST['price'];
+        $location = $_POST['location'];
+        $zipCode = $_POST['zipCode'];
+        $description = $_POST['description'];
+        $emailInput = $_POST['emailInput'];
+        $conditionInput = $_POST['conditionInput'];
+        $phoneNumberInput = $_POST['phoneNumberInput'];
+        $categoryInput = $_POST['categoryInput'];
+        // $imageFile = $_POST['fileToUpload']; //Currently there is no image file support atm (S3 In the Future?)
+        if (isset($_POST['deliveredInput'])) {
+            $deliveryAvailable = 'Yes';
+        } else {
+            $deliveryAvailable = 'No';
+        }
+        $verifiedUser = getUser($emailInput, $phoneNumberInput); 
+        if ($verifiedUser) {
+        $sellerId = $verifiedUser['userId'];
+        insertProduct($postingTitle, $price, $description, $sellerId, $location, $zipCode, $categoryInput, $deliveryAvailable, $conditionInput);
+        } else {
+            echo 'Please Login Before Trying to Post an Item!'; 
+            //This case should (almost) never occur... atm I plan on having users registering or logging in before making a post.
+        }
```

Comment:
You would be surprised how many people bookmark pages they shouldn't :)

This is out of scope for this PR, but one thing that LACRM does in this case is redirect people to the login page. The flow looks a little like this:
- Someone not logged in visits /workspace
- We redirect them to /login.php?RedirectAfterLogin=/workspace. They see the login screen like normal.
- If they successfully log in, they automatically end up on the page they were trying to access.

Again, don't have to do this now, just wanted to share one approach.

Reply:
One thing you should change: you should probably call [exit()](https://www.php.net/manual/en/function.exit.php) at the end of this block to make sure the rest of the page doesn't end up printing (unless you want it to).